src/dsymbol/conversion/first.d

```diff
@@ -152,6 +152,7 @@ final class FirstPass : ASTVisitor
 			pushFunctionScope(dec.functionBody, semanticAllocator,
 					dec.name.index + dec.name.text.length);
 			scope (exit) popScope();
+            includeParameterSymbols = true;
```

Comment:
this should not be here (it effectively now makes the constructor parameter useless, as parameters would always be included)

If it's never a problem, consider instead removing the symbol completely, deprecating the ctor and stripping all the `ifs` that checked on it